app/models/invenio_rdm_record_converter.rb

```diff
@@ -174,9 +174,9 @@ def invenio_metadata
       "resource_type": resource_type(@generic_file.resource_type.shift),
       "creators": @generic_file.creator.map{ |creator| build_creator_contributor_json(creator) },
       "title": @generic_file.title.first,
-      "additional_titles": additional(category: "title", array: @generic_file.title),
+      "additional_titles": format_additional("title", "alternative-title", @generic_file.title.drop(1)),
```

Comment:
the drop(1) is a little annoying, but it seemed like the fastest way to handle this and works well enough.